packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -1006,33 +1010,48 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
      */
 
     function executeSpamDeletion(uint256 proposalId) external nonReentrant() {
-        require(spamDeletionProposals[proposalId].executed == false, "Proposal already executed");
```

Comment:
we hit the contract size limit so I removed some requires that will never really be hit.